qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -83,865 +60,81 @@ def _to_int(value):
-
-    def delete_prep_template(self, study, user, callback):
-        """Delete the selected prep template
-
-        Parameters
-        ----------
-        study : Study
-            The current study object
-        user : User
-            The current user object
-        callback : function
-            The callback function to call with the results once the processing
-            is done
-        """
-        prep_template_id = int(self.get_argument('prep_template_id'))
-        prep_id = PrepTemplate(prep_template_id).raw_data
-
-        try:
-            PrepTemplate.delete(prep_template_id)
-            msg = ("Prep template %d has been deleted" % prep_template_id)
-            msg_level = "success"
-            prep_id = None
-        except Exception as e:
-            msg = ("Couldn't remove prep template: %s" % str(e))
-            msg_level = "danger"
-
-        callback((msg, msg_level, 'prep_template_tab', prep_id, None))
-
-    def delete_preprocessed_data(self, study, user, callback):
-        """Delete the selected preprocessed data
-
-        Parameters
-        ----------
-        study : Study
-            The current study object
-        user : User
-            The current user object
-        callback : function
-            The callback function to call with the results once the processing
-            is done
-        """
-        ppd_id = int(self.get_argument('preprocessed_data_id'))
-
-        try:
-            PreprocessedData.delete(ppd_id)
-            msg = ("Preprocessed data %d has been deleted" % ppd_id)
-            msg_level = "success"
-            ppd_id = None
-        except Exception as e:
-            msg = ("Couldn't remove preprocessed data %d: %s" %
-                   (ppd_id, str(e)))
-            msg_level = "danger"
-
-        callback((msg, msg_level, 'preprocessed_data_tab', ppd_id, None))
-
-    def delete_processed_data(self, study, user, callback):
-        """Delete the selected processed data
-
-        Parameters
-        ----------
-        study : Study
-            The current study object
-        user : User
-            The current user object
-        callback : function
-            The callback function to call with the results once the processing
-            is done
-        """
-        pd_id = int(self.get_argument('processed_data_id'))
-
-        try:
-            ProcessedData.delete(pd_id)
-            msg = ("Processed data %d has been deleted" % pd_id)
-            msg_level = "success"
-            pd_id = None
-        except Exception as e:
-            msg = ("Couldn't remove processed data %d: %s" %
-                   (pd_id, str(e)))
-            msg_level = "danger"
-
-        callback((msg, msg_level, 'processed_data_tab', pd_id, None))
+class StudyIndexHandler(BaseHandler):
+    def study_prep_proxy(self, study):
+        """Proxies expected json from the API for existing prep templates"""
```

Comment:
Can we bring back the standard numpy doc strings?  It makes it a bit easier to understand what's going on here.